persiancalendar/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,40 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
     <string name="app_name">Persian Calendar</string>
+    <string name="pcv_shanbe">Sat</string>
+    <string name="pcv_yekshanbe">Sun</string>
+    <string name="pcv_doshanbe">Mon</string>
+    <string name="pcv_seshanbe">Tue</string>
+    <string name="pcv_chaharshanbe">Wed</string>
+    <string name="pcv_panjshanbe">Thu</string>
+    <string name="pcv_jome">Fri</string>
+    <string name="pcv_one">1</string>
+    <string name="pcv_two">2</string>
+    <string name="pcv_three">3</string>
+    <string name="pcv_four">4</string>
+    <string name="pcv_five">5</string>
+    <string name="pcv_six">6</string>
+    <string name="pcv_seven">7</string>
+    <string name="pcv_eight">8</string>
+    <string name="pcv_nine">9</string>
+    <string name="pcv_farvardin">Farvardin</string>
```

Comment:
No this is weird. finglish is not a thing!
Since now you've put these strings, in resources. we can just have the basic set of Farsi names. and the developers can override these strings in their apps according to their needs.
But "Farvardin" is not something I like to have in this library